### PR TITLE
Add filterClassName

### LINKED ISSUE #259: Add css class to filter input
I was expecting something like `filterClassName` but it is not implemented. In the end I injected a class: 
> $('.reactable-filter-input').addClass('form-control');

which is ugly and not reliable.

May i know if there's any better ways to achieve this? I'm new to react so do let me know if there's a better alternative. Otherwise are you good with a PR on implementing a `filterClassName`?

### PR DESCRIPTION
**Pull request by shinnyx**
Fixes #259

### CODE REVIEW

File: tests/reactable_test.jsx [hunk 1]
```diff
@@ -411,6 +411,10 @@ describe('Reactable', function() {
                 ReactTestUtils.Simulate.keyUp($filter[0]);
             });
 
+            it('add new-class', function() {
```
Comment (glittershark) [inline tests/reactable_test.jsx]: Can you make this "adds the filterClassName to the filterer"?

**Comment by glittershark** [comment]
What's up with the changes to the built test file?

**Comment by glittershark** [comment]
Also, I'd rather not have a bunch of unrelated whitespace changes in here, or the changes to test.html

**Comment by shinnyx** [comment]
@glittershark I'm not sure about the changes on build test file, FYI I ran the test with `npm test`. Perhaps there's some configurations that I'm missing?

**Comment by glittershark** [comment]
try a `grunt buildOnce` and then amend the commit. Also, please keep this branch squashed to one commit

**Comment by shinnyx** [comment]
@glittershark the build test file changed in https://github.com/glittershark/reactable/commit/8a8486669cfb976448539573a290c05978b23447 so I will just stick with this.

Just FYI when I run `grunt buildOnce` I'll get this `[BABEL] Note: The code generator has deoptimised the styling of "tests/reactable_test.jsx" as it exceeds the max of "100KB".`. Adding `{ compact: false }` will ignore the warning.

**Comment by shinnyx** [comment]
@glittershark hi, is there anything else blocking for the PR to be merged?

**Comment by StefanYohansson** [comment]
> try a grunt buildOnce and then amend the commit. Also, please keep this branch squashed to one commit

@glittershark about squash thing, github now implements squash and merge.
https://help.github.com/articles/about-pull-request-merge-squashing/

**Comment by glittershark** [comment]
I prefer to keep merge commits, but have the branch I'm merging from have logical, isolated commits. This particular change only reasonably should take one commit, but that's not the case for all PRs

**Comment by glittershark** [comment]
Sorry for the delay. This is fine given that it's babel that's mucking up the built test file

**Pull request merged**